examples/hello-scroll.js

```diff
@@ -3,13 +3,14 @@
 const Max7219 = require('..')
 const controllerCount = 4
 
-const m = new Max7219({ device: '/dev/spidev0.0', controllerCount, flip: 'vertical' })
+const m = new Max7219({ device: '/dev/spidev0.0', controllerCount, flip: 'none' })
```

Comment:
we can probably just remove the flip here, the default should be `none`.

Reply:
All examples updated to change to resetAll() and to remove default option flags.